terraform/modules/concourse_worker/security_groups.tf

```diff
@@ -29,6 +19,16 @@ resource "aws_security_group_rule" "worker_ucfs_github_outbound_https" {
   cidr_blocks       = [var.github_cidr_block]
 }
 
+resource "aws_security_group_rule" "worker_lb_out_ssh" {
+  description       = "outbound traffic to web nodes from worker nodes via lb"
+  from_port         = 2222
+  protocol          = "tcp"
+  security_group_id = aws_security_group.worker.id
+  to_port           = 2222
+  type              = "egress"
+  cidr_blocks       = var.vpc.aws_subnets_private.*.cidr_block
```

Comment:
can this be replaced with a security group?

Reply:
As above.